VCDMExplainer.md

```diff
@@ -111,9 +111,21 @@ Here is an example of usage of the ID property in a credential, using the DID sc
 ```
 
 ## Tricky Design Choices
-* JOSE vs JWT - see [Issue 93](https://github.com/w3c/vc-data-model/issues/93)
-* JSON vs JSON-LD
-* Selective disclosure as a requirement see [Issue 224](https://github.com/w3c/vc-data-model/issues/224) for example
+
+### JSON vs JSON-LD
```

Comment:
I think we should remove this heading and just leave "Support for JWT" below.

Reply:
Ok, I am fine with that. Updated the PR.